### PR TITLE
Skip schema tests using supportedOperations method

### PR DESCRIPTION
**Pull request by MXPOL**

### CODE REVIEW

File: packages/velo-external-db/test/storage/schema_provider.spec.js [hunk 1]
```diff
@@ -104,45 +108,61 @@ describe('Schema API', () => {
...
+                    await expect(env.schemaProvider.addColumn(ctx.collectionName, { name: f, type: 'datetime', subtype: 'timestamp' })).rejects.toThrow(CannotModifySystemField)
+                })
         })
 
-        if (shouldNotRunOn(['BigQuery', 'Google-Sheet'], name)) {
-            test('drop column on a an existing collection', async() => {
+        test('drop column on a an existing collection', async() => {
+            const neededOperations = [REMOVE_COLUMN]
+
+            if ( neededOperations.every(op => env.schemaOperations.includes(op)) ) {
```
Comment (noam-almog) [inline packages/velo-external-db/test/storage/schema_provider.spec.js:123]: can this be a generic function somewhere ?

File: packages/velo-external-db/test/storage/schema_provider.spec.js [hunk 2]
```diff
@@ -104,45 +108,61 @@ describe('Schema API', () => {
...
-                        })
+                .forEach(async f => {
+                    await expect(env.schemaProvider.addColumn(ctx.collectionName, { name: f, type: 'datetime', subtype: 'timestamp' })).rejects.toThrow(CannotModifySystemField)
+                })
         })
 
-        if (shouldNotRunOn(['BigQuery', 'Google-Sheet'], name)) {
-            test('drop column on a an existing collection', async() => {
+        test('drop column on a an existing collection', async() => {
+            const neededOperations = [REMOVE_COLUMN]
```
Comment (noam-almog) [inline packages/velo-external-db/test/storage/schema_provider.spec.js:121]: why do we need this variable ?

File: packages/velo-external-db/test/storage/schema_provider.spec.js [hunk 3]
```diff
@@ -85,16 +86,19 @@ describe('Schema API', () => {
...
-            test('add column on a an existing collection', async() => {
-                await env.schemaProvider.create(ctx.collectionName, [])
-
-                await env.schemaProvider.addColumn(ctx.collectionName, { name: ctx.columnName, type: 'datetime', subtype: 'timestamp' })
 
-                await expect( env.schemaProvider.describeCollection(ctx.collectionName) ).resolves.toEqual( hasSameSchemaFieldsLike([{ field: ctx.columnName, type: 'datetime' }]))
-            })
-        }
+        test('add column on a an existing collection', async() => {
+            const neededOperations = [ADD_COLUMN]
```
Comment (Idokah) [inline packages/velo-external-db/test/storage/schema_provider.spec.js:91]: I think requiredOperations will be a better name

File: packages/velo-external-db/test/storage/schema_provider.spec.js [hunk 4]
```diff
@@ -85,16 +86,19 @@ describe('Schema API', () => {
...
 
+            if ( neededOperations.every(op => env.schemaOperations.includes(op)) ) {
+                await env.schemaProvider.create(ctx.collectionName, [])
+                await env.schemaProvider.addColumn(ctx.collectionName, { name: ctx.columnName, type: 'datetime', subtype: 'timestamp' })
+                await expect( env.schemaProvider.describeCollection(ctx.collectionName) ).resolves.toEqual( hasSameSchemaFieldsLike([{ field: ctx.columnName  }]))
+            }
+            else {
+                expect(true).toBe(true)
```
Comment (Idokah) [inline packages/velo-external-db/test/storage/schema_provider.spec.js:99]: Maybe export this to a function as well. 
`passTest` or something similar that includes that assertion.

**Pull request merged**